app/controllers/application_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def deualt 
+    programmme
```

Comment:
__[development][Misspell]__

"programmme" is a misspelling of "programme"


<!-- Comment by [Sider](https://sider.review) -->